src/nanoemoji/write_font.py

```diff
@@ -138,22 +138,24 @@ class ColorGenerator(NamedTuple):
 assert _COLOR_FORMAT_GENERATORS.keys() == set(config._COLOR_FORMATS)
 
 
-def _ufo(family: str, upem: int, keep_glyph_names: bool = False) -> ufoLib2.Font:
+def _ufo(config: FontConfig) -> ufoLib2.Font:
     ufo = ufoLib2.Font()
-    ufo.info.familyName = family
+    ufo.info.familyName = config.family
     # set various font metadata; see the full list of fontinfo attributes at
-    # http://unifiedfontobject.org/versions/ufo3/fontinfo.plist/
-    ufo.info.unitsPerEm = upem
+    # https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/#generic-dimension-information
+    ufo.info.ascender = config.ascent
+    ufo.info.descender = config.descent
+    ufo.info.unitsPerEm = config.upem
 
     # Must have .notdef and Win 10 Chrome likes a blank gid1 so make gid1 space
     ufo.newGlyph(".notdef")
     space = ufo.newGlyph(".space")
     space.unicodes = [0x0020]
-    space.width = upem
+    space.width = config.upem
```

Comment:
should you also scale the default width by the user-provided transform? otherwise it won't be centered anymore.

Also, maybe the width itself could be a CLI/config option since it is an interesting metric like ascent/descent

Reply:
+1 to width config; will add.

Reply:
If the width is a config option I'm less sure it should be subject to transform; thoughts?

Reply:
don't transform if provided explicitly. the user can do the math

Reply:
agreed and so implemented